api/server/routes/ask.js

```diff
@@ -53,6 +53,12 @@ router.post('/', async (req, res) => {
     }
   }
 
+  if (id) {
```

Comment:
Not sure why but conversations are not updating in the Convo NavBar once submitted or when another convo is clicked (expected behavior), whether in a new chat or resubmitted. Maybe id is being kept the same throughout api calls? 

I noticed this in a new chat, and the same is true for resubmissions. This was the start of the convo:
![image](https://user-images.githubusercontent.com/110412045/224579765-f1477713-23c3-444d-963d-1f1e476b8e7b.png)
I had not refreshed and sent another message but it got split into a separate convo:
![image](https://user-images.githubusercontent.com/110412045/224579802-489110c0-0a1e-4b0f-9866-d07d6825140f.png)

Reply:
fixed